src/xrpld/app/tx/detail/DeleteAccount.cpp

```diff
@@ -316,12 +347,21 @@ DeleteAccount::doApply()
     auto src = view().peek(keylet::account(account_));
     assert(src);
 
-    auto dst = view().peek(keylet::account(ctx_.tx[sfDestination]));
+    auto const dstID = ctx_.tx[sfDestination];
+    auto dst = view().peek(keylet::account(dstID));
     assert(dst);
 
     if (!src || !dst)
         return tefBAD_LEDGER;
 
+    if (ctx_.view().rules().enabled(featureDepositAuth) &&
```

Comment:
Don't need to check `DepositAuth` in the new code since it's enabled, right?

Reply:
All the other code have it, and we still have tests  that disable it, so better let it be